tools_docker/README.md

```diff
@@ -57,4 +57,17 @@ To build image
 
     cd tools_docker/
     docker build -t algoo/tracim:latest .
-    
\ No newline at end of file
+
+You can build with specific branch
+
+    cd tools_docker/
+    docker build --build-arg BRANCH="<branch_name>" -t algoo/tracim:<version_name> .
+
+Ex: `docker build --build-arg BRANCH="feature/new_app" -t algoo/tracim:test_branch .`
+    
+You can also build image with specific tag (This build is make just with necessary files:)
+
+    cd tools_docker/
+    docker build --build-arg TAG="-b <tag_name> --single-branch" -t algoo/tracim:<tag_name> .
+    
+Ex: `docker build --build-arg TAG="-b release_02.00.00 --single-branch" -t algoo/tracim:release_02.00.00 .`
```

Comment:
not good: `TAG= ` must contain a tag name only, not parameters for the command line.

`TAG="-b release_02.00.00 --single-branch"` => wrong because -b and --signle-branch are not TAG name

`TAG=release_02.00.00` is ok

Reply:
by the way, if I give a branch and a tag, then you checkout 2 differents versions of the code, which seems to bu useless... the right way to do this is probably to "build the git command" before to run all the commands, then to run the right git command which you prepared earlier.

Reply:
I dont really understand "build the git command"?
Ok for naming of ARG, I tried to find better naming.

Reply:
with command `-b --single-branch` git clone just commit about this tag and just this branch. Its not possible with this to go to anther branch.

Reply:
Done.

Reply:
@tracim now its not possible to use ARG TAG and ARG BRANCH in same time. If you try this, docker build stop with message.